src/feedback/vue/submitFeedback.vue

```diff
@@ -82,9 +86,14 @@ export default defineComponent({
             this.isSubmitting = true
             console.log('Submitting feedback...')
             // identifier to help us (internally) know that feedback came from either CodeWhisperer or AWS Toolkit
+            let userComment =
+                this.feedbackName === 'CodeWhisperer' ? 'CodeWhisperer onboarding: ' + this.comment : this.comment
+            // info to help us (internally) know the jobId and clientId for transform by Q job runs
+            this.extraCodeTransformData['jobId'] = transformByQState.getJobId()
+            this.extraCodeTransformData['clientId'] = await getClientId(globals.context.globalState)
```

Comment:
Feedback is supposed to be anonymous...

Reply:
I've been told that we are permitted to include this by multiple others. Will follow up on this.

Reply:
@damntrecky

Reply:
If we are calling an API, wont the users credentials be logged when calling whatever feedback API is in place?

For the clientside logging, whats the difference to adding `clientId` which is added in every Telemetry event. The id that matches the users IDE download? Not their AWS Account Id. 

There are several emit telemetry events around this feedback call already `telemetry.feedback_result.emit({ result: 'Succeeded' })`. This call is not anonymous right? `clientId` will be present.

Where is the public documentation on this API? It appears codewhisper API's are [not public](https://docs.aws.amazon.com/codewhisperer/latest/userguide/useful-apis.html)? I'm used to having public everything from previous services so my following questions probably don't apply.

Do we legally state that this information is completely anonymous? And why? I only found a small snippet in a readme and the vue file itself.

If feedback is to remain anonymous, we need to present information to our stakeholders so they can make an appropriate decision.

<img width="807" alt="Screenshot 2023-12-18 at 9 16 02 AM" src="https://github.com/aws/aws-toolkit-vscode/assets/38665452/ff399dd6-1609-4b17-9fe6-210e83c3bcef">
![Screenshot 2023-12-18 at 9 23 35 AM](https://github.com/aws/aws-toolkit-vscode/assets/38665452/9d57f716-d398-46cc-b6d8-26fc8f8d383e)

![Screenshot 2023-12-18 at 9 19 12 AM](https://github.com/aws/aws-toolkit-vscode/assets/38665452/71a8a444-dda5-4f97-95ac-8932677f5a5e)

Reply:
This code snippet is outdated so its hard to place the context of where this was. Not sure if its in the `.emit` call or the submission for the API itself.

Reply:
I think I found some documentation -> https://docs.aws.amazon.com/toolkit-for-jetbrains/latest/userguide/compliance-validation.html

Reply:
data protection docs for IntelliJ -> https://docs.aws.amazon.com/toolkit-for-jetbrains/latest/userguide/data-protection.html

Don't mention anonymous and same for VSCode -> https://docs.aws.amazon.com/toolkit-for-vscode/latest/userguide/data-protection.html